lib/location.py

```diff
@@ -125,54 +130,54 @@ def ip2location(ip_addr):
     # if IP2LOCATION_KEY is not set, do not query,
     # because the query wont be processed anyway
     if not IP2LOCATION_KEY:
-        return None, None, None
+        return None
     try:
         r = requests.get(
-            'http://api.ip2location.com/?ip=%s&key=%s&package=WS3'
+            'http://api.ip2location.com/?ip=%s&key=%s&package=WS5'
```

Comment:
WS5 is 50% more expensive, and is useless for us, because we already have location -> lat,long conversion implementation

Reply:
This means that ip2location will not be able to cache the lat,long at this step, and we'd have to cache differently. How else do you suggest we get that information?

Reply:
ip2location makes only one step in the conversion pipeline:
it converts IP-address into "location" (a tuple city,province,country),
which is then converted into a (lat,long) pair with `geolocator` (external service).

![1](https://user-images.githubusercontent.com/3875145/99293223-a0766b80-2842-11eb-8a8e-85ca325cc3b4.png)

If we would process only ip addresses, we could stay with this approach,
but we need to convert location queries too (wttr.in/London),
so we need the `geolocator` step anyway

Reply:
In that case, we should probably remove `lat` and `long` capture from the other two services, and let the pair be resolved later by `geolocator`. Do you think `geolocator` should have its own cache?